addon-test-support/ember-test-helpers/wait.js

```diff
@@ -1,7 +1,46 @@
 export {
-  default,
   _setupAJAXHooks,
   _setupPromiseListeners,
   _teardownAJAXHooks,
   _teardownPromiseListeners,
 } from '@ember/test-helpers/settled';
+
+import { waitUntil, getSettledState } from '@ember/test-helpers';
+
+/**
+  Returns a promise that resolves when in a settled state (see `isSettled` for
+  a definition of "settled state").
+
+  @private
+  @deprecated
+  @returns {Promise<void>} resolves when settled
+*/
+export default function settled() {
```

Comment:
I think this should be called `wait()`, not `settled()`

Reply:
DOH! Indeed!